be/src/storage/storage_engine.cpp

```diff
@@ -1199,7 +1199,12 @@ Status StorageEngine::get_next_increment_id_interval(int64_t tableid, size_t num
         auto st = _get_remote_next_increment_id_interval(alloc_params, &alloc_result);
 
         if (!st.ok() || alloc_result.status.status_code != TStatusCode::OK) {
-            return Status::InternalError("auto increment allocate failed");
+            std::string err_msg;
+            for (auto msg : alloc_result.status.error_msgs) {
```

Comment:
```suggestion
            for (auto& msg : alloc_result.status.error_msgs) {
```
use reference to reduce copy